bin/run-versioned-tests.sh

```diff
@@ -96,4 +100,4 @@ then
 fi
 export NR_LOADER=./esm-loader.mjs
 
-time $C8 ./node_modules/.bin/versioned-tests $VERSIONED_MODE -i 2 --all --strict --samples $SAMPLES $JOBS_ARGS ${directories[@]}
+time $C8 ./node_modules/.bin/versioned-tests $VERSIONED_MODE --print $OUTPUT_MODE -i 2 --all --strict --samples $SAMPLES $JOBS_ARGS ${directories[@]}
```

Comment:
Note: this change results in local `npm run versioned...` _also_ using the "quiet" mode. Personally, I prefer that. But I can do some more work to default local runs to "pretty" mode if desired.

Reply:
can you actually default to pretty and then in our ci workflows specify quiet?